packages/standard-linter/index.js

```diff
@@ -11,6 +11,13 @@ module.exports = {
   rules: {
     "@typescript-eslint/no-floating-promises": "error",
     "no-console": "error",
+    "max-classes-per-file": "off",
+    "class-methods-use-this": "off",
+    "no-await-in-loop": "off",
+    // functions and classes are going to be hoisted in runtime
+    "no-use-before-define": ["error", { functions: false, classes: false }],
```

Comment:
even if functions are hoisted during runtime, wouldn't it be better to have the functions aligned to just before where they are being used? would be easier to trace the code since we know how and when they are used, one block after another

Reply:
Agree that a top to bottom flow is better but this is more for complex scenarios where we cannot align the function flow nicely.

Reply:
spoke about this, this was more of the scenario for util functions used by the class (and class goes above since it's easier to know the class is being defined). I'll resolve this comment